src/decision/test/gps-manager-test.cpp

```diff
@@ -49,206 +49,6 @@ TEST(GpsManager, parseWaypoints){
-}
-
-TEST(GpsManager, convertToRobotsPrespective_origin_rotated_left_straight_ahead){
-    // Waypoint is 10 meters directly ahead
-    Waypoint wp;
-    sensor_msgs::NavSatFix nsfix;
-    nsfix.latitude = 49.254226;
-    nsfix.longitude = -123.240825;
-    wp.lat = 49.25431591;
-    wp.lon = -123.240825;
-    double origin_heading = 0.5 * M_PI;
-
-    geometry_msgs::Point converted_waypoint = GpsManager::convertToRobotsPerspective(wp, nsfix, origin_heading);
-    EXPECT_NEAR(0, converted_waypoint.x, 0.005);
-    EXPECT_NEAR(-10, converted_waypoint.y, 0.005);
-}
-
-TEST(GpsManager, convertToRobotsPrespective_origin_rotated_left_directly_left){
-    // Waypoint is 10 meters directly ahead
-    Waypoint wp;
-    sensor_msgs::NavSatFix nsfix;
-    nsfix.latitude = 49.254226;
-    nsfix.longitude = -123.240825;
-    wp.lat = 49.254226;
-    wp.lon = -123.24096275;
-    double origin_heading = 0.5 * M_PI;
-
-    geometry_msgs::Point converted_waypoint = GpsManager::convertToRobotsPerspective(wp, nsfix, origin_heading);
-    EXPECT_NEAR(10, converted_waypoint.x, 0.005);
-    EXPECT_NEAR(0, converted_waypoint.y, 0.005);
-}
-
-TEST(GpsManager, convertToRobotsPrespective_origin_rotated_left_directly_behind){
-    // Waypoint is 10 meters directly behind
-    Waypoint wp;
-    sensor_msgs::NavSatFix nsfix;
-    nsfix.latitude = 49.25431591;
-    nsfix.longitude = -123.240825;
-    wp.lat = 49.254226;
-    wp.lon = -123.240825;
-    double origin_heading = 0.5 * M_PI;
-
-    geometry_msgs::Point converted_waypoint = GpsManager::convertToRobotsPerspective(wp, nsfix, origin_heading);
-    EXPECT_NEAR(0, converted_waypoint.x, 0.005);
-    EXPECT_NEAR(10, converted_waypoint.y, 0.005);
-}
-
-TEST(GpsManager, convertToRobotsPrespective_rotated_45_straight_ahead){
-    // Waypoint is 10 meters directly ahead
-    // Origin rotation is Pi/2
-    Waypoint wp;
-    sensor_msgs::NavSatFix nsfix;
-    nsfix.latitude = 49.254226;
-    nsfix.longitude = -123.240825;
-    wp.lat = 49.25431591;
-    wp.lon = -123.240825;
-    double origin_heading = M_PI/4;
-
-    geometry_msgs::Point converted_waypoint = GpsManager::convertToRobotsPerspective(wp, nsfix, origin_heading);
-    EXPECT_NEAR(10 * cos(origin_heading), converted_waypoint.x, 0.005);
-    EXPECT_NEAR(-10 * cos(origin_heading), converted_waypoint.y, 0.005);
-}
-
-TEST(GpsManager, convertToRobotsPrespective_rotated_60_straight_ahead){
-    // Waypoint is 10 meters directly ahead
-    // Origin rotation is 60 degrees
-    Waypoint wp;
-    sensor_msgs::NavSatFix nsfix;
-    nsfix.latitude = 49.254226;
-    nsfix.longitude = -123.240825;
-    wp.lat = 49.25431591;
-    wp.lon = -123.240825;
-    double origin_heading = 60 * (M_PI/180);
-
-    geometry_msgs::Point converted_waypoint = GpsManager::convertToRobotsPerspective(wp, nsfix, origin_heading);
-    EXPECT_NEAR(10 * cos(origin_heading), converted_waypoint.x, 0.005);
-    EXPECT_NEAR(-10 * cos(M_PI/2 - origin_heading), converted_waypoint.y, 0.005);
-}
-
```

Comment:
It's sad that these tests are vanishing, because tests make me happy, but on the other hand, we're removing them because we're just relying on `TF`'s to handle all this properly for us now, so I'm OK with that...